sphinx-build dependency missing

I've installed sphinx-autobuild through pip on Fedora 29.
After running `sphinx-autobuild -z <codesourcepath> "<docspath>" "<outputpath>"` I'm getting:
```
+--------- manually triggered build ---------------------------------------------
Traceback (most recent call last):
  File "/usr/bin/sphinx-autobuild", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/sphinx_autobuild/__init__.py", line 312, in main
    builder.build()
  File "/usr/lib/python2.7/site-packages/sphinx_autobuild/__init__.py", line 175, in build
    subprocess.Popen(args, stdout=slave)
  File "/usr/lib64/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
make: *** [Makefile:18: livehtml] Error
```
By further investigation I found out that I'm missing sphinx-build executable. I believe this should be marked as dependency. Or at least improve the error message.

* the `args` contained `['sphinx-build', '-v', '-T', '<valid path>', '<valid path>']`

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,6 +25,7 @@ classifiers = [
 home-page = "https://github.com/GaretJax/sphinx_autobuild"
 license = "MIT"
 requires = [
+  "sphinx",
   "watchdog",
   "livereload",
 ]
@@ -34,4 +35,4 @@ sphinx-autobuild = "sphinx_autobuild.__main__:main"
 
 [tool.flit.metadata.requires-extra]
 test = ["mock", "pytest", "pytest-cov"]
-docs = ["sphinx"]
+docs = []

--- a/src/sphinx_autobuild/sphinx.py
+++ b/src/sphinx_autobuild/sphinx.py
@@ -61,7 +61,7 @@ class SphinxBuilder(object):
         sys.stdout.write("-" * (81 - len(pre)))
         sys.stdout.write("\n")
 
-        args = ["sphinx-build"] + self._args
+        args = [sys.executable, '-m', 'sphinx'] + self._args
         if pty:
             master, slave = pty.openpty()
             stdout = os.fdopen(master)

